exercises/concept/lasagna/.docs/hints.md

```diff
@@ -4,18 +4,23 @@
 
 - You will need to define [functions][functions] and use [arithmetic operators][arithmetics].
 
-## 1. Calculate the preparation time in minutes
+## 1. Store the expected bake time in a constant
+
+- Define a [const][const] to store this value.
+
+## 2. Calculate the preparation time in minutes
 
 - Use the [`times` arithmetic operator][arithmetics] to multiply the argument by `2`.
```

Comment:
If it wouldn't be too much information, we could consider including how Julia can do implicit multiplication here too (maybe as an aside)

```julia
c = 4
2c == 2*c == 8 # evaluates as true
```